scripts/js/lib/api/generateToc.ts

```diff
@@ -74,9 +74,15 @@ function getModulesAndItems(
   );
   const items = resultsWithName.filter(
     (result) =>
-      result.meta.apiType === "class" ||
-      result.meta.apiType === "function" ||
-      result.meta.apiType === "exception",
+      result.meta.apiType &&
+      [
+        "class",
+        "function",
+        "exception",
+        "method",
+        "property",
+        "attribute",
+      ].includes(result.meta.apiType),
```

Comment:
Is this what makes the Sampler and Estimator appear in the TOC?

Reply:
Yes, exactly. In the upcoming PR to add `..py:data::`, we'll add `data` here as well which fixes the remaining orphan pages we have in API docs 🚀